packages/@unimodules/react-native-adapter/android/build.gradle

```diff
@@ -1,4 +1,5 @@
 buildscript {
+  ext.kotlin_version = '1.3.61'
```

Comment:
Imho we should leverage `ext.kotlinVersion` which can be set by the root project and then use it like this:
```
safeExtGet('kotlinVersion', '1.3.61')
```